tests/core/pyspec/eth2spec/test/merge/block_processing/test_process_execution_payload.py

```diff
@@ -13,31 +18,178 @@ def run_execution_payload_processing(spec, state, execution_payload, valid=True,
     If ``valid == False``, run expecting ``AssertionError``
     """
 
-    pre_exec_header = state.latest_execution_payload_header.copy()
-
     yield 'pre', state
     yield 'execution', {'execution_valid': execution_valid}
     yield 'execution_payload', execution_payload
 
+
+    spec.verify_execution_state_transition_ret_value = execution_valid
```

Comment:
This is dirty and unsafe. The best way of implementing it would be in creating a copy of spec with required changes, e.g. `spec = spec.copy(verify_execution_state_transition_ret_value = execution_valid)` in this particular case.

Reply:
#2398 resolves this